include/OpenSpaceToolkit/Astrodynamics/EventCondition/COECondition.hpp

```diff
@@ -43,122 +44,129 @@ using ostk::astro::trajectory::State;
 class COECondition
 {
    public:
+    /// @brief                  Semi-Major Axis based constructor
+    ///
+    /// @param                  [in] aCriterion An enum indicating the criterion used to determine the Event Condition
+    /// @param                  [in] aFrameSPtr A frame in which the Element is to be computed
+    /// @param                  [in] aTarget A Target
+    /// @param                  [in] aGravitationalParameter A gravitational parameter
+
     static RealCondition SemiMajorAxis(
```

Comment:
nit: Think it's worth keeping the original factory function that takes a Length as well to avoid breaking backwards compatibility here? Not sure where all these are used, but could do a similar thing as `AngularCondition` above. Your call

Reply:
They're marginally used in some notebooks, I think it's easier to update it than support it for backwards compatability IMO